### PR TITLE
Fix RTL icon placement

### PR DESCRIPTION
**Pull request by yosilevy**
## Proposed change

RTL fixes to placements
We should be using ha-list-item anywhere we have icons. I will work on it later on but these are important.

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/automation/action/ha-automation-action.ts [hunk 1]
```diff
@@ -279,6 +279,11 @@ export default class HaAutomationAction extends LitElement {
           margin-bottom: 16px;
           scroll-margin-top: 48px;
         }
+        ha-svg-icon[slot="icon"] {
```
Comment (bramkragten) [inline src/panels/config/automation/action/ha-automation-action.ts:282]: Why this one? It doesn't have this in ltr now? (also, make the selector more specific)

File: src/panels/config/automation/trigger/ha-automation-trigger.ts [hunk 2]
```diff
@@ -280,6 +280,11 @@ export default class HaAutomationTrigger extends LitElement {
           margin-bottom: 16px;
           scroll-margin-top: 48px;
         }
+        ha-svg-icon[slot="icon"] {
```
Comment (bramkragten) [inline src/panels/config/automation/trigger/ha-automation-trigger.ts:283]: same

**Comment by yosilevy** [comment]
The comments were resolved in latest commit. @bramkragten

**Comment by bramkragten** [comment]
> The comments were resolved in latest commit. @bramkragten

Looks like you didnt push that?

**Comment by yosilevy** [comment]
There was no commit because as I explained above there is nothing to fix. See comment with yellow and pink markings... @bramkragten

**Comment by bramkragten** [comment]
> There was no commit because as I explained above there is nothing to fix. See comment with yellow and pink markings... @bramkragten

I'm not seeing it? 🤷‍♂️

**Comment by yosilevy** [comment]
It's here: https://github.com/home-assistant/frontend/pull/14019#discussion_r991340922

Here's what I wrote:
Because we have ha-svg-icon in the top level (that's the icon) element which the one that needs fixing BUT also in the items. The items are fixed by ha-list-item so we must not apply more styles.
See image below.
Yellow needs fixing, pink doesn't (ran on old code so still shows mwc-list-item)
image

**Comment by yosilevy** [comment]
@bramkragten see previous comment. Thanks

**Review by bramkragten — DISMISSED** [review]

**Comment by bramkragten** [comment]
> It's here: [#14019 (comment)](https://github.com/home-assistant/frontend/pull/14019#discussion_r991340922)
> 
> Here's what I wrote: Because we have ha-svg-icon in the top level (that's the icon) element which the one that needs fixing BUT also in the items. The items are fixed by ha-list-item so we must not apply more styles. See image below. Yellow needs fixing, pink doesn't (ran on old code so still shows mwc-list-item)

So we are fixing a bug in `mwc-button`? We should preferably create a `ha-button` that fixes this, instead of having the parent fix it everywhere.

**Comment by yosilevy** [comment]
@bramkragten Done

**Review by bramkragten — DISMISSED** [review]

**Comment by Misiu** [comment]
what about other `mwc-button` usages? If now we have `ha-button` shouldn't we use that component everywhere?
Maybe that's for separate PR.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**